src/NServiceBus.Core/DelayedDelivery/TimeoutManager/ExpiredTimeoutsPoller.cs

```diff
@@ -84,7 +84,7 @@ public void Dispose()
                 if (nextRetrieval > DateTime.UtcNow)
                 {
                     // ReSharper disable once MethodSupportsCancellation
-                    await Task.Delay(1000);
+                    await Task.Delay(1000).ConfigureAwait(false);
```

Comment:
OT: Wish MS would have made this the default :( The new `virtual` by default Vietnam..